<launch>     
      <rosparam file="$(find jetson_camera)/config/motor_params.yaml" 
      command="load" />      
      <node name="motor_controller"           
      pkg="jetson_camera"           
      type="motor_controller.py"           
      output="screen"           
      respawn="false" />      
      <node name="encoder_controller"           
      pkg="jetson_camera"           
      type="encoder_controller.py"           
      output="screen"           
      respawn="false" />      
</launch>